res/values/strings.xml

```diff
@@ -834,12 +835,11 @@
     <string name="ephemeral_timer_weeks_by_you">You set disappearing messages timer to %1$s weeks.</string>
     <!-- %1$s will be replaced by the number of weeks (always >1) the timer is set to, %2$s will be replaced by name and address of the contact -->
     <string name="ephemeral_timer_weeks_by_other">Disappearing messages timer set to %1$s weeks by %2$s.</string>
-    <string name="protection_enabled_by_you">You enabled chat protection.</string>
-    <!-- %1$s will be replaced by name and address of the contact -->
-    <string name="protection_enabled_by_other">Chat protection enabled by %1$s.</string>
-    <string name="protection_disabled_by_you">You disabled chat protection.</string>
-    <!-- %1$s will be replaced by name and address of the contact -->
-    <string name="protection_disabled_by_other">Chat protection disabled by %1$s.</string>
+    <string name="chat_protection_broken">%1$s sent a message from another device.</string>
```

Comment:
```suggestion
    <!-- this may be shown instead of the chat's input field, with the buttons as "More Info" or "OK" -->
    <string name="chat_protection_broken">%1$s sent a message from another device.</string>
```

Reply:
```suggestion
    <!-- this may be shown instead of the chat's input field, with buttons "More Info" and "OK" -->
    <string name="chat_protection_broken">%1$s sent a message from another device.</string>
```